internal/resources/provisioner/provisioner_data_source_test.go

```diff
@@ -84,12 +83,11 @@ func verifyProvisionerDataSource(name string) resource.TestCheckFunc {
 	}
 }
 
-// TODO: Add the meta check after TMC-54016 fix.
-// func metaDataSourceAttributeCheck(dataSourceName, resourceName string) []resource.TestCheckFunc {
-//	return []resource.TestCheckFunc{
-//		resource.TestCheckResourceAttrPair(dataSourceName, "meta.0.description", resourceName, "meta.0.description"),
-//		resource.TestCheckResourceAttrPair(dataSourceName, "meta.0.labels.key1", resourceName, "meta.0.labels.key1"),
-//		resource.TestCheckResourceAttrPair(dataSourceName, "meta.0.labels.key2", resourceName, "meta.0.labels.key2"),
-//		resource.TestCheckResourceAttrSet(dataSourceName, "meta.0.uid"),
-//	}
-//}
+func metaDataSourceAttributeCheck(dataSourceName, resourceName string) []resource.TestCheckFunc {
+	return []resource.TestCheckFunc{
+		resource.TestCheckResourceAttrPair(dataSourceName, "provisioners.0.meta.0.description", resourceName, "meta.0.description"),
```

Comment:
By specifying  `provisioners.0.meta.0.description` does this limit the check to only one provisioners meta info even if there are multiple?

Reply:
In the data source config used in the test we are reading one provisioner only. Hence, we are checking for the same one here